### DISCUSSION TITLE
How to give multiple users access to the same peers?

### DISCUSSION THREAD

**Discussion by MTRNord** [netbirdio/netbird#626] [Q&A / Support] [answered]
Hi I want to use the selfhosted version within my family. I have users managed using keycloak. How can I make it that all users are able to access all the peers?

PS: I dont see any invite user button

**Comment by MTRNord** [comment, marked as answer]
Ok using a wrong email address made it work. I wish there was a better way with keycloak though and not only with auth0

**Reply by helmut72** [reply]
How do you enabled the invite user button? I created a user with a non-existent mail address, logged into netbird, but still doesn't see any invite button. Also using self hosted version with keycloak. Thanks.

**Reply by braginini** [reply]
The invite is not available for the self-hosted because all user management is done via your own IDP, like Keycloack. 
Just add a user in Keycloak and they should be added automatically to the network once logged in.

**Reply by MTRNord** [reply]
> Just add a user in Keycloak and they should be added automatically to the network once logged in.

thats just the half truth as I learned.

Netbird seems to auth check based on the email.

assuming you have a user with a@emailA.com and a user b with b@emailB.com. this would not work. Since the env var `NETBIRD_MGMT_SINGLE_ACCOUNT_MODE_DOMAIN` only allows to set a single domain. So my solution was to have an incorrect email of `userb@emailB.com`  for user b instead of using a valid email address. If you just add the oidc user and use any other email domain than the one set in the env var you end up with having no user b in netbird.
